lib/main.dart

```diff
@@ -112,6 +112,15 @@ class _RandomWordsState extends State<RandomWords> {
           ),
         ],
       ),
+      floatingActionButton: FloatingActionButton(
+        onPressed: () {
+          setState(() {
+            _suggestions.insert(0, WordPair.random());
```

Comment:
It would be more intuitive if WordPair.random() was assigned to a variable e.g. "newStartupName", so that it could be called from anywhere in the code